src/ims/store/mysql/_queries.py

```diff
@@ -213,6 +213,67 @@
         select max(NUMBER) from INCIDENT where EVENT = ({query_eventID})
         """,
     ),
+    incidents=Query(
+        "look up incidents for event",
+        f"""
+        select
+            i.NUMBER,
+            i.CREATED,
+            i.PRIORITY,
+            i.STATE,
+            i.SUMMARY,
+            i.LOCATION_NAME,
+            i.LOCATION_CONCENTRIC,
+            i.LOCATION_RADIAL_HOUR,
+            i.LOCATION_RADIAL_MINUTE,
+            i.LOCATION_DESCRIPTION,
+            i.EVENT,
+            (
+                select json_arrayagg(it.NAME)
```

Comment:
`json_arrayagg` is too cool for me, so I'll just assume this is sane. 🙃

Reply:
haha, yeah, I just learned about it myself. It's relatively new. It takes a bunch of things and turns them into a JSON array (i.e. a string that can be parsed as a JSON array)